src/fshtypes/FshCode.ts

```diff
@@ -1,14 +1,20 @@
 import { FshEntity } from './FshEntity';
 import { Coding, CodeableConcept, Quantity } from '../fhirtypes';
+import { fshifyString } from './common';
 
 export class FshCode extends FshEntity {
   constructor(public code: string, public system?: string, public display?: string) {
     super();
   }
 
   toString(): string {
-    const str = `${this.system ?? ''}#${/\s/.test(this.code) ? `"${this.code}"` : this.code}`;
-    return this.display ? `${str} "${this.display}"` : str;
+    let codeToShow = this.code;
+    const fshifiedCode = fshifyString(this.code);
```

Comment:
I'm having trouble understanding why the code needs to be `fshified` and placed in the quotes unless it has a space. I've found that I can do stuff like this:
```
Profile: O
Parent: Observation
* code = #strange\"code
```
and now the preprocessed output comes out as:
```
Profile: O
Parent: Observation
Id: O
* code = #"strange\\\"code"
```
which also looks correct, but doesn't seem necessary. Are there cases I'm missing where the code does not have a space, but does still need to be placed in quotes?

Reply:
I could have sworn I ran into a case with just `\` and `"` that needed to be quoted, but now I can't seem to reproduce it. So, I think you are correct that the code only needs to be `fshified` and quoted if it contains spaces.